### PR TITLE
HMT-37 Under Construction Placeholder

### PR DESCRIPTION
**Pull request by hannagracec**
Added a placeholder to the landing page that says "under construction" with icons that link to the Instagram and LinkedIn for people to stay updated. 
image

### CODE REVIEW

File: src/components/PagePlaceholder.jsx
Comment (hannagracec) [reply src/components/PagePlaceholder.jsx:9]: Okay, thanks for letting me know I'll change that right now!

**Review by ideen1 — DISMISSED** [review]

**Comment by ideen1** [comment]
Please merge this after #4 has been merged. Otherwise looks great!

**Review by ideen1 — APPROVED** [review]

**Pull request merged**
